package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "_s",
+  "version": "1.0.0",
+  "scripts": {
+    "css-build": "npm run css-lint && npm run css-compile && npm run css-prefix && npm run css-format && npm run css-fix",
+    "css-lint": "stylelint style.css",
+    "css-compile": "node-sass sass/style.scss style.css --output-style expanded --indent-type tab --indent-width 1",
+    "css-prefix": "postcss style.css -u autoprefixer -o style.css --no-map",
+    "css-format": "stylefmt style.css",
+    "css-fix": "stylelint style.css --fix",
+    "scss-lint": "stylelint sass/**/*.scss",
+    "js-lint": "eslint js/**",
+    "js-fix": "eslint js/** --fix"
+  },
+  "devDependencies": {
+    "autoprefixer": "^7.1.2",
+    "browserslist": "^2.1.5",
+    "eslint": "^4.1.1",
+    "eslint-config-wordpress": "^2.0.0",
+    "node-sass": "^4.5.3",
+    "postcss-cli": "^4.1.0",
+    "stylefmt": "^6.0.0",
+    "stylelint": "^7.12.0",
+    "stylelint-config-wordpress": "^11.0.0"
+  },
+  "stylelint": {
+    "defaultSeverity": "warning",
+    "extends": "stylelint-config-wordpress",
+    "rules": {
+      "max-line-length": [ 80, {
+        "ignorePattern": ["/\b(Description:).*\n/", "/https?:\/\/[0-9,a-z]*.*/"]
```

Comment:
This `['/(https?://[0-9,a-z]*.*)|(^description\\:.+)|(^tags\\:.+)/i']` should do the trick. Though we may not need it if we you the latest WordPress config.
https://github.com/WordPress-Coding-Standards/stylelint-config-wordpress/pull/151